dir.props

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-
+  <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
+  
```

Comment:
Added to enable inclusion of dir.props file located in directory above repo root. This enables coordinated changes to build configurations of multiple dotnet projects build side by side. It is used in [corefx](https://github.com/dotnet/corefx/blob/master/dir.props#L3), [standard](https://github.com/dotnet/standard/blob/master/dir.props#L3), [core-setup](https://github.com/dotnet/core-setup/blob/master/dir.props#L3), [corert](https://github.com/dotnet/corert/blob/master/dir.props#L2) and other repos.